mongodb/devhub
# Community Champions profile page - layout UI [3/x]

## Links:

-   [JIRA Ticket](https://jira.mongodb.org/browse/DEVHUB-784)
-   [Staging Link](https://docs-mongodbcom-staging.corp.mongodb.com/CI/devhub/sophia.li/DEVHUB-784-layout-ui-links/community-champions/chris-dellaway/)
- [Staging Link 2](https://docs-mongodbcom-staging.corp.mongodb.com/CI/devhub/sophia.li/DEVHUB-784-layout-ui-links/community-champions/arkadiusz-borucki/)
- [Staging Link 3](https://docs-mongodbcom-staging.corp.mongodb.com/CI/devhub/sophia.li/DEVHUB-784-layout-ui-links/community-champions/dani-monteiro/)

## Description:

-   Created the "Blogs & Publications" and "Social" sections as well as Certifications!

## Testing

-  Check that the UI for links and certifications match the mockups on figma
-  Check that the text is green when hovering over the links
-  Check that the links open in a new tab

## What types of outside events need to happen for this PR?

-   [ ] Release Note Update (only for deployments)
